libs/pxt-python/pxt-python-helpers.ts

```diff
@@ -119,11 +119,6 @@ namespace _py {
         return str;
     }
 
-    export function py_string_replace(str: string, oldString: string, newString: string, count?: number): string {
```

Comment:
Just to confirm, my recollection here is that this is only ever used as a shim / is not available to be called directly from the editor, right? (reason for asking being whether a user project could have a direct dependency on the function being available)

Reply:
Good point. It could be called directly from the editor (in JS). However in this case it isn't implemented, it's just the identity function, so it'd be a very odd dependency for someone to have.

Reply:
But you're point stands: in the future, we can't get rid of these functions once people start using them. (When they have implementations.)